docs/src/site/twiki/EntitySLAAlerting.twiki

```diff
@@ -0,0 +1,57 @@
+---++Entity SLA Alerting
+
+Falcon supports SLA in feed and process.
```

Comment:
Per @sepreece's suggestion, this should read - Falcon supports monitoring SLA of feed and process.